internal/exec/stages/disks/disks.go

```diff
@@ -212,7 +212,7 @@ func (s stage) createRaids(config types.Config) error {
 			args = append(args, util.DeviceAlias(string(dev)))
 		}
 
-		if err := s.Logger.LogCmd(
+		if _, err := s.Logger.LogCmd(
```

Comment:
`LogCmd` was changed to return the exit code in addition to any errors:
https://github.com/coreos/ignition/pull/363/files#diff-d0beabc8e56f67d1ac3c08a5b05ad846R135

This was necessary so that when Ignition checks for existing users via `chroot /sysroot id <name>` it can differentiate between the user not existing (exit code 1) and other errors (127 for a missing binary, 125 for the path is invalid).

In this specific invocation, the code is not used, hence the added `_, `